Content.Client/Chemistry/UI/ReagentCardControl.xaml

```diff
@@ -0,0 +1,42 @@
+<Control xmlns="https://spacestation14.io"
+         xmlns:gfx="clr-namespace:Robust.Client.Graphics;assembly=Robust.Client"
+         xmlns:controls="clr-namespace:Content.Client.UserInterface.Controls"
+         xmlns:customControls="clr-namespace:Content.Client.Administration.UI.CustomControls">
+    <BoxContainer Name="MainContainer"
+                  Orientation="Horizontal"
+                  SetWidth="160">
+        <PanelContainer Name="ColorPanel"
+                        VerticalExpand="True"
+                        SetWidth="7"
+                        Margin="0 1 0 0" />
+        <Button Name="MainButton"
+                HorizontalExpand="True"
+                VerticalExpand="True"
+                StyleClasses="ButtonSquare"
+                Margin="-1 0 0 0">
+            <BoxContainer Orientation="Horizontal" HorizontalExpand="True">
+                <BoxContainer Orientation="Vertical"
+                              VerticalExpand="True"
+                              HorizontalExpand="True"
+                              Margin="-4 0 0 0">
+                    <Label Name="ReagentNameLabel"
+                           Access="Public"
```

Comment:
Avoid making controls public, write helper methods on ReagentCardControl for this instead.

Reply:
Yeah I was mostly just copying previous code and added those without thinking really.